### ISSUE TITLE
How to implement a custom “<audio>” tag?

### ISSUE THREAD

**Issue by FunnySaltyFish** [noties/Markwon#454]
* **Markwon version**: newest   

Hello, I want to build a translation app using Markown, and thanks for your astonishing work, it can show complex translation result. However, I'm not sure how to show an `audio` tag to implement `speaking the result aloud` function. The tag might be like:
```html
<audio src="https://fanyi.baidu.com/gettts?lan=en&text=hello&spd=5&source=web"/>
```
And the markdown might be:

```markdown
**Examples:**
1. example one <audio src="audio_url_of_example_1"></audio>
2. example two <audio src="audio_url_of_example_2"></audio>
3. ...
```

It might be rendered as an image or a button, and can set the `onClick` event so I can call some functions to speak the audio. It will be better to be an animatable widget so users can know whether the audio is playing or not. I've checked the examples and still don't know where to start. 

Thanks!

**Comment by zhouzhuo810** [comment]
@FunnySaltyFish 我也遇到这个问题了，你解决了没

**Comment by FunnySaltyFish** [comment]
nope, just like what you saw, no body answered. If you have a solution, it would be very helpful that sharing here,

**Comment by winiymissl** [comment]
> 没有，就像你看到的一样，没有人回答。如果你有解决方案，分享到这里会很有帮助。

在htmlPlugin启动的时候删除掉原本支持的<audio>标签，添加自定义标签，这个可以支持自定义绘制
